api/server/graph.cc

```diff
@@ -1066,7 +1064,12 @@ void Graph::NicConfigPacketTracePath(const app::Nic &nic,
     Graph::GraphNic *g_nic = FindNic(nic);
     FILE *n_pcap_file;
     std::string name;
-    if (nic.packet_trace_path == update_path) {
+    std::string upath;
+    size_t found;
+
+    found = update_path.find_last_of("/");
+    upath = update_path.substr(0,found);
+    if (nic.packet_trace_path == upath) {
```

Comment:
I think you need to evaluate the full path, I mean it's okay to change path from /tmp/bibi.pcap to /home/titi/bibi.pcap.
and your code ignore that case